#goal definition
geometry_msgs/Twist uturn_vel
float32 elapsed_time
geometry_msgs/Pose2D start_point
geometry_msgs/Pose2D end_point
---
#result definition
int32 state
---
#feedback
int32 state
